expected-consensus.md

```diff
@@ -321,6 +321,8 @@ All valid blocks generated in a round form a `TipSet` that participants will att
 
 The first condition implies that all blocks in a TipSet were mined at the same height (remember that height refers to block height as opposed to ticket round). This rule is key to helping ensure that EC converges over time. While multiple new blocks can be mined in a round, subsequent blocks all mine off of a TipSet bringing these blocks together. The second rule means blocks in a TipSet are mined in a same round.
 
+The blocks in a tipset are canonically ordered by ticket.
```

Comment:
I had thought we settled on making the order not matter at all?

Reply:
This line refers to the logical ordering, the order of processing in consensus state transitions, rather than the "syntactic" requirement (or not) for ordering in the block header. Can I clarify that in the text more?

For the block header, I was waiting for your call in #285 whether to order by CID or not at all.

Reply:
Ah, okay. I think that what we want to say is that there is no ordering enforced for tipsets, but during state computation, we apply the transitions of blocks in a tipset ordered by their tickets.